### DISCUSSION TITLE
Dataset Structure For Training Key Information Extraction

### DISCUSSION THREAD

**Discussion by raditharch** [open-mmlab/mmocr#1607] [Q&A]
Hi, well done for this amazing work!

I am wondering about structure dataset wildreceipt. I am trying to training Key Information Extraction with new dataset but using pretrained model sdmgr_unet16_60e_wildreceipt.py.

Inside folder image_files there is folder Image_1, Image_2, Image_...., Image_n and inside folder Image_1 there is folder again with number but i can't find correlation this number with anything and inside folder number there is 20 image. Why did you split image into 20 and put it inside folder of folder? 

This structure of wildreceipt
└── wildreceipt
──├── class_list.txt
──├── dict.txt
──├── image_files
──────└── Image_1
──────────└── 0
─────────────└── img.jpeg # 20 image
──────└── Image_5
──────────└── 0
─────────────└── img.jpeg #  5 image
──────────└── 1
─────────────└── img.jpeg # 15 image
──────└── Image_12
──────────└──10
─────────────└── img.jpeg # 20 image
──────└── Image_n
──├── test.txt
──└── train.txt

Thank you.

**Comment by gaotongxiao** [comment]
Interesting question. See if @cuhk-hbsun has any comments
